cmd/util_ref.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+)
+
+func fromStrings(namespace, name string) string {
+	return fmt.Sprintf("%s:%s", namespace, name)
+}
+
+func parse(ref string) (namespace string, name string) {
```

Comment:
Name needs to be changed to reflect what this function does e.g. `parseProjectID`